arviz/plots/autocorrplot.py

```diff
@@ -51,20 +51,20 @@ def plot_autocorr(
         automatically inferred.
     figsize: tuple
         Figure size. If None it will be defined automatically.
-        Note this is not used if ax is supplied.
+        Note this is not used if ``ax`` is supplied.
     textsize: float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
-        on figsize.
+        on ``figsize``.
     labeller : labeller instance, optional
-        Class providing the method `make_label_vert` to generate the labels in the plot titles.
+        Class providing the method ``make_label_vert`` to generate the labels in the plot titles.
         Read the :ref:`label_guide` for more details and usage examples.
     ax: numpy array-like of matplotlib axes or bokeh figures, optional
         A 2D array of locations into which to plot the densities. If not supplied, Arviz will create
         its own array of plot areas (and return it).
     backend: str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
     backend_config: dict, optional
-        Currently specifies the bounds to use for bokeh axes. Defaults to value set in rcParams.
+        Currently specifies the bounds to use for bokeh axes. Defaults to value set in :class:`arviz.RcParams`.
     backend_kwargs: dict, optional
         These are kwargs specific to the backend being used. For additional documentation
         check the plotting method of the backend.
```

Comment:
This is missing the links to the matplotib/bokeh methods where these kwargs are passed eventually

Reply:
Same as [plot_hdi](https://arviz-devs.github.io/arviz/api/generated/arviz.plot_hdi.html), backend_kwargsbool, optional?